docs/framework/wcf/wcf-test-client-wcftestclient-exe.md

```diff
@@ -8,6 +8,9 @@ Windows Communication Foundation (WCF) Test Client (WcfTestClient.exe) is a GUI
 
 You can typically find the WCF Test Client (WcfTestClient.exe) in the following location: `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE` - Community may be one of "Enterprise", "Professional" or "Community" depending on which level of Visual Studio is installed.
 
+## Prerequisite 
+This WcfTestClient.exe only available by selecting the Windows Communication Foundation (under Development activities of VS installer)
```

Comment:
The https://docs.microsoft.com/en-us/dotnet/framework/wcf/using-the-wcf-development-tools article notes this with the following note block. I've edited your contribution to match that note.

```suggestion

[!NOTE] Starting with Visual Studio 2017, the WCF development tools are not installed by default. In order to use these features, you must ensure the Windows Communication Foundation component is selected in the Visual Studio installer.
```